modules/300-prometheus/images/alerts-receiver/src/config.go

```diff
@@ -22,21 +22,35 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
-	"k8s.io/client-go/kubernetes"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/rest"
 )
 
 const (
-	reconcileTime = 5 * time.Minute
-	nameSpace     = "d8-monitoring"
+	appName          = "prometheus"
+	reconcileTime    = 1 * time.Minute
+	resolveTimeout   = 5 * time.Minute
+	contextTimeout   = 10 * time.Second
+	severityLabel    = "severity_level"
+	summaryLabel     = "summary"
+	descriptionLabel = "description"
 )
 
+var GVR = schema.GroupVersionResource{
+	Group:    "deckhouse.io",
+	Version:  "v1alpha1",
+	Resource: "clusteralerts",
+}
+
+var livenessOK = true
```

Comment:
I think, we need to use [atomic.Bool](https://pkg.go.dev/sync/atomic#Bool) here

And it is not config. I think you can move it into `alerts-receiver.go` We use this variable only there

Reply:
yep, thanks!